### ISSUE TITLE
btAssert from HingeJoint.setAngularOnly(true)

### ISSUE THREAD

**Issue by stephengold** [stephengold/Minie#20]
How to reproduce:  add `joint.setAngularOnly(true);` to the `HelloDoor` app and run with a debug native library.
On Linux I get:
```text
Feb 04, 2022 3:50:01 PM com.jme3.audio.openal.ALAudioRenderer initOpenAL
INFO: Audio max auxiliary sends: 2
Debug_Libbulletjme version 12.7.1 initializing
warning btCollisionDispatcher::needsCollision: static-static collision!
java: /home/sgold/Git/Libbulletjme/src/main/native/bullet3/BulletDynamics/ConstraintSolver/btSequentialImpulseConstraintSolver.cpp:1265: void btSequentialImpulseConstraintSolver::convertJoint(btSolverConstraint*, btTypedConstraint*, const btTypedConstraint::btConstraintInfo1&, int, int, const btContactSolverInfo&): Assertion `fsum > SIMD_EPSILON' failed.
```

The btAssert may be unnecessary, since there's code below it to handle the failure case...

**Comment by stephengold** [comment]
Wielded the Hammer: https://github.com/stephengold/Libbulletjme/commit/ebdee715d87826a4159be400b206cddcc3f98df9

Should be solved in next release.

**Comment by stephengold** [comment]
Solved in "master" branch at 65de6404. I verified the fix using `HelloDoor`.

**Comment by stephengold** [comment]
The fix is included in Minie v4.7.1 .
